### PR TITLE
Various fixes (lower-bounds checker edition, round 4)

### PR DESCRIPTION
**Pull request by kit-ty-kate**

### CODE REVIEW

File: packages/conf-rust-2018/conf-rust-2018.1/opam [hunk 1]
```diff
@@ -6,21 +6,25 @@ bug-reports: "https://github.com/ocaml/opam-repository/issues"
 license: "MIT"
 extra-files: ["test.rs" "md5=639e04c270fef8589636e0416761a67b"]
```
Comment (avsm) [inline packages/conf-rust-2018/conf-rust-2018.1/opam:7]: This change to conf-rust looks unrelated to lower bounds.

Comment (kit-ty-kate) [reply packages/conf-rust-2018/conf-rust-2018.1/opam]: I moved this into its own PR
